src/scripts/main.js

```diff
@@ -1 +1,56 @@
 'use strict';
+
+const body = document.body;
+
+function createNotification(message, classValue) {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.innerText = message;
+  notification.classList = classValue;
+  body.append(notification);
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  body.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(Error('First promise was rejected'));
+  }, 3000);
+});
+
+function waitFor(evnt, number) {
+  return new Promise((resolve) => {
+    document.addEventListener(evnt, () => {
+      resolve(`${number} promise was resolved`);
+    });
+  });
+}
+
+firstPromise
+  .catch(error => {
+    createNotification(error, 'warning');
```

Comment:
Change the order of 'catch' and 'then' to avoid skipping error handling for the firstPromise.